### DISCUSSION TITLE
✅ Sticky Note Widget

### DISCUSSION THREAD

**Discussion by cynomyso** [elementor/elementor#17672] [Feature Request]
I would love to have an element that could be dropped into the interface somehow that ONLY appeared in the editor and was used for leaving notes for yourself or the next editor. I know this could KIND of be done with something like responsive hide settings or display conditions, but it seems like a dedicated widget had no ability to be displayed on a published page would be better. Some kind of preset color/border settings to set it aside as a note, like a sticky note would set it apart visually from other page elements. Thanks!

**Comment by rhud** [comment]
looks like this might be possible soon - https://forum.elementor.com/product-updates/dev-introducing-notes-work-collaboratively-with-clients-and-colleagues-directly-within-elementor-13697

**Comment by WebsiteDons** [comment]
You could use the hook `add_action( 'elementor/documents/register_controls' )` as shown at https://developers.elementor.com/docs/editor/page-settings-panel/ . This will allow adding any field type along with custom coding for various display and functionality.

This example uses a simple textarea, however with some JS and repeater field, it could be more complex to store date, author, user restriction, checkbox to show when job is done etc.

```php
add_action('elementor/documents/register_controls', function($document) 
{
  if( !$document instanceof \Elementor\Core\DocumentTypes\PageBase || !$document::get_property('has_elements') )  {
    return;
  }

  $document->start_controls_section('document_settings',[
    'tab' => \Elementor\Controls_Manager::TAB_SETTINGS,
  ]);

  $document->add_control('_admin_note',[
    'label' => 'Notepad',
    'type' => \Elementor\Controls_Manager::TEXTAREA
  ]);

  $document->end_controls_section();
});
```

![2022-05-09 01_52_50](https://user-images.githubusercontent.com/42153624/167348973-119fee79-0b38-4112-9f06-2cd57f9c51a8.jpg)

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.7.0! 🥳 

You can now use the [Notes feature](https://elementor.com/help/notes-feature/) to collaborate and communicate with other users, and even your customers!

✅ Feel free to check it out!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
